packages/web-components/src/radio/radio.styles.ts

```diff
@@ -0,0 +1,182 @@
+import { css } from '@microsoft/fast-element';
+import { display } from '@microsoft/fast-foundation';
+import {
+  borderRadiusMedium,
+  colorCompoundBrandForeground1,
+  colorCompoundBrandForeground1Pressed,
+  colorCompoundBrandStroke,
+  colorCompoundBrandStrokeHover,
+  colorCompoundBrandStrokePressed,
+  colorNeutralForeground1,
+  colorNeutralForeground2,
+  colorNeutralForeground3,
+  colorNeutralForegroundDisabled,
+  colorNeutralStrokeAccessible,
+  colorNeutralStrokeAccessibleHover,
+  colorNeutralStrokeAccessiblePressed,
+  colorStrokeFocus1,
+  colorStrokeFocus2,
+  fontFamilyBase,
+  fontSizeBase300,
+  fontWeightRegular,
+  lineHeightBase300,
+  spacingHorizontalS,
+  spacingHorizontalXS,
+  spacingVerticalS,
+  spacingVerticalXS,
+} from '../theme/design-tokens.js';
+
+/** Radio styles
+ * @public
+ */
+export const styles = css`
+  ${display('inline-flex')}
+
+  :host {
+    align-items: center;
+    cursor: pointer;
+    display: flex;
+    flex-direction: row;
+    outline: none;
+    position: relative;
+    user-select: none;
+  }
+
+  .label {
+    color: ${colorNeutralForeground3};
+    cursor: pointer;
+    font-family: ${fontFamilyBase};
+    font-size: ${fontSizeBase300};
+    font-weight: ${fontWeightRegular};
+    line-height: ${lineHeightBase300};
+    padding: ${spacingVerticalS} ${spacingHorizontalS} ${spacingVerticalS} ${spacingHorizontalXS};
+  }
+
+  .label__hidden {
+    display: none;
+    visibility: hidden;
+  }
+
+  .control {
+    align-items: center;
+    border: 1px solid ${colorNeutralStrokeAccessible};
+    border-radius: 50%;
```

Comment:
Just a question - is this hardcoded or should this be a token value? I can see an argument for both

Reply:
Good catch.. I can't see a scenario where Fluent would ever make the radio anything but a perfect circle but they do have a `borderRadiusCircular` so probably best to go ahead and use it.